README.md

```diff
@@ -87,6 +87,8 @@ number -> int
   * **start_pos** - the byte index into the string at which the match starts, inclusive
   * **pos** - the byte index into the string at which the match ends, exclusive
 
+If you're using Vim, [syntax highlight plugin](https://github.com/rhysd/vim-rustpeg) is available.
```

Comment:
Thank you for review. It's helpful becuase I'm not a native English user. I'll choose latter.